ci-operator/config/openshift/knative-client/openshift-knative-client-release-next.yaml

```diff
@@ -22,8 +13,5 @@ tests:
     cluster_profile: aws
 resources:
   '*':
-    limits:
-      memory: 4Gi
     requests:
-      cpu: 100m
-      memory: 200Mi
```

Comment:
If the resources are insufficient, make them larger, but I don't think this should be removed entirely.

Reply:
I am requesting 2Gi of memory (line no: 17).

You mean, restore the `limits` section ?
I removed the limits following the discussion on testplatform channel.
/cc @stevekuznetsov

Reply:
Not having limits is good -- we don't want to deal with OOMkilled pods unless we have to